pkgs/development/tools/bazelisk/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
Since it has a `go.mod` file you can use `buildGoModule`: https://kalbas.it/2019/03/17/announcing-the-new-golang-infrastructure-buildgomodule/

Reply:
Thanks, that does look nice! I don't feel too bad not knowing about `buildGoModule`, since it wasn't even [PR'd](https://github.com/NixOS/nixpkgs/pull/57080) until three days after I submitted this one :-) Exciting to see the stream of improvements always moving things forward. I'll re-work this.